src/DynamoCoreWpf/UI/Prompts/CrashPrompt.xaml.cs

```diff
@@ -38,10 +43,15 @@ public CrashPrompt(DynamoViewModel dynamoViewModel)
             txtOverridingText.Text = string.Format(Wpf.Properties.Resources.CrashPromptDialogCrashMessage, productName);
         }
 
-        public CrashPrompt(CrashPromptArgs args, DynamoViewModel dynamoViewModel)
+        public CrashPrompt(CrashPromptArgs args, DynamoViewModel dynamoViewModel, PackageLoader packageLoader)
```

Comment:
API break ⚠️

Reply:
hmm purely UI code here, I understand we are never clear about our API scope but would like to argue that we should be able to break our own UI layer logic.. especially about dialogs not top Dynamo view layer

Reply:
Instead of passing it in, get it by `dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader;`